pages/doc/logging_log_alerts.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Manage Alerts for Logs
+tags: [logs]
+sidebar: doc_sidebar
+permalink: logging_log_alerts.html
+summary: Learn how you can create, snooze and delete a logs alert.
+---
+
+{% include important.html content="Our service retains logs for 30 days during the Logs free trial or retains logs for 7 days when on the freemium subscription! To retain logs for a longer period, contact your account representative or [technical support](wavefront_support_feedback.html#support)."%}
+
+You can create alerts for your logs data and see the firing events of an alert.
+
+<!-- Add this after email notification and remove the line above
+
+You can create alerts for your logs data and get email notifications when the alert conditions are met.
+-->
+
+{{site.data.alerts.note}}
+<ul>
+    <li>
+        You need the <b>Logs</b> permission to view the logs alerts.
+    </li>
+    <li>
+        You need the <b>Alerts</b> and <b>Logs</b> permissions to create and manage logs alerts.
+    </li>
+    <p>The steps to add roles and permissions differ for <a href="wavefront_release_notes.html#announcement">Onboarded and Original subscriptions</a>. See add permissions details for <a href="csp_users_roles.html#create-edit-or-delete-a-custom-role">Onboarded subscriptions</a> and <a href="users_roles.html">Original subscriptions</a>.</p>
```

Comment:
The steps to add roles and permissions differ for onboarded and original subscriptions. For details, see how to manage roles, groups, and permissions for <a href="csp_users_roles.html#create-edit-or-delete-a-custom-role">onboarded subscriptions</a> and <a href="users_roles.html">original subscriptions</a>.